<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWidgetTitle</class>
 <widget class="QWidget" name="QWidgetTitle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>675</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>675</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>675</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>欢迎登录绿建项目管理系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/res/upda.ico</normaloff>:/res/upda.ico</iconset>
  </property>
  <widget class="myWidgetBk" name="widget_tbtn_close" native="true">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>0</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QToolButton" name="tbtn_close">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>220</y>
     <width>401</width>
     <height>151</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>华文新魏</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>密    码：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comboBox">
      <property name="minimumSize">
       <size>
        <width>260</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>260</width>
        <height>32</height>
       </size>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 16pt &quot;华文新魏&quot;;
background-color: rgb(218, 218, 218);
</string>
      </property>
      <property name="insertPolicy">
       <enum>QComboBox::InsertAtBottom</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit">
      <property name="minimumSize">
       <size>
        <width>260</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>260</width>
        <height>32</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(218, 218, 218);
font: 16pt &quot;华文新魏&quot;;</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>华文新魏</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>用户名：</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>comboBox</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>lineEdit</zorder>
  </widget>
  <widget class="MyPushButton" name="btn_register">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>370</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文新魏</family>
     <pointsize>20</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;华文新魏&quot;;</string>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/register.png</normaloff>:/res/register.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="MyPushButton" name="btn_login">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>370</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文新魏</family>
     <pointsize>20</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 20pt &quot;华文新魏&quot;;</string>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/login.svg</normaloff>:/res/login.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="myWidgetBk" name="widget_tbtn_min" native="true">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>0</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QToolButton" name="tbtn_min">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myWidgetBk</class>
   <extends>QWidget</extends>
   <header location="global">mywidgetbk.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MyPushButton</class>
   <extends>QPushButton</extends>
   <header location="global">mypushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
